examples/src/main/java/ai/djl/examples/inference/ObjectDetectionWithTensorflowSavedModel.java

```diff
@@ -0,0 +1,250 @@
+import ai.djl.modality.cv.output.Rectangle;
+import ai.djl.modality.cv.util.NDImageUtils;
+import ai.djl.ndarray.NDArray;
+import ai.djl.ndarray.NDList;
+import ai.djl.ndarray.NDManager;
+import ai.djl.ndarray.types.DataType;
+import ai.djl.ndarray.types.Shape;
+import ai.djl.repository.zoo.Criteria;
+import ai.djl.repository.zoo.ModelZoo;
+import ai.djl.repository.zoo.ZooModel;
+import ai.djl.training.util.ProgressBar;
+import ai.djl.translate.Batchifier;
+import ai.djl.translate.TranslateException;
+import ai.djl.translate.Translator;
+import ai.djl.translate.TranslatorContext;
+import ai.djl.util.JsonUtils;
+import com.google.gson.annotations.SerializedName;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Scanner;
+import java.util.concurrent.ConcurrentHashMap;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * An example of inference of object detection using saved model from TensorFlow 2 Detection Model
+ * Zoo.
+ *
+ * <p>Tested with EfficientDet, SSD MobileNet V2, Faster RCNN Inception Resnet V2 downloaded from <a
+ * href="https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/tf2_detection_zoo.md">here</a>
+ *
+ * <p>See this <a
+ * href="https://github.com/awslabs/djl/blob/master/examples/docs/object_detection_with_tensorflow_saved_model.md">doc</a>
+ * for information about this example.
+ */
+public final class ObjectDetectionWithTensorflowSavedModel {
+
+    private static final Logger logger =
+            LoggerFactory.getLogger(ObjectDetectionWithTensorflowSavedModel.class);
+
+    private ObjectDetectionWithTensorflowSavedModel() {}
+
+    public static void main(String[] args) throws IOException, ModelException, TranslateException {
+        DetectedObjects detection = ObjectDetectionWithTensorflowSavedModel.predict();
+        logger.info("{}", detection);
+    }
+
+    public static DetectedObjects predict() throws IOException, ModelException, TranslateException {
+        if (!"TensorFlow".equals(Engine.getInstance().getEngineName())) {
+            throw new UnsupportedOperationException("This test only for TensorFlow");
+        }
+
+        Path imageFile = Paths.get("src/test/resources/dog_bike_car.jpg");
+        Image img = ImageFactory.getInstance().fromFile(imageFile);
+
+        String modelUrl =
+                "http://download.tensorflow.org/models/object_detection/tf2/20200711/ssd_mobilenet_v2_320x320_coco17_tpu-8.tar.gz";
+
+        Criteria<Image, DetectedObjects> criteria =
+                Criteria.builder()
+                        .optApplication(Application.CV.OBJECT_DETECTION)
+                        .setTypes(Image.class, DetectedObjects.class)
+                        .optModelUrls(modelUrl)
```

Comment:
FYI: optModelUrls() is preferred way to specify url, the system properties will impact the jvm globally and not thread-safe if you change it at runtime.